### ISSUE TITLE
html plugin: json to table rendering error

### ISSUE THREAD

**Issue by waldenn** [spencermountain/wtf_wikipedia#352]
For any article with a table, I see this error message:

> TypeError: can't convert undefined to object   [table.js:6](https://github.com/spencermountain/wtf_wikipedia/blob/cbb3ea359f2a5259a1a84d360155ba7318e736fd/plugins/html/src/table.js#L7):10

```js
  Object.keys(this[0]).forEach(k => {
    if (/^col[0-9]/.test(k) !== true) {
      html += '    <td>' + k + '</td>\n'
    }
  })
```

Not sure yet what is causing "this" to be undefined in some cases. There is data in the JSON structure though, when one filters out the "undefined" values.

**Comment by spencermountain** [comment]
hey thanks, this is fixed now in plugin-html `0.2.1`.
let me know if you see anything else
cheers

**Comment by waldenn** [comment]
Will do, thanks Specer!
